docker-compose-integration-test.yml

```diff
@@ -26,6 +26,8 @@ services:
   rust-sdk:
     image: experimenter:integration-tests
     env_file: .env
+    environment:
+      - DEBIAN_FRONTEND=0
```

Comment:
Just some context, I am using this as a flag to signify a container without having to add more variables to both images.

Reply:
Can we use an env var to just specifiy the kinto URL instead?